OATControl/MainWindow.xaml

```diff
@@ -1,4 +1,4 @@
-﻿<Controls:MetroWindow x:Class="OATControl.MainWindow"
+<Controls:MetroWindow x:Class="OATControl.MainWindow"
```

Comment:
The changes to this file are not right. It looks like you may have used the Designer to insert an extra row, which it does in a stupid way.... Can you send me a screenshot of what this achieves? I'll send you the Xaml to use.